.github/workflows/tests.yml

```diff
@@ -14,6 +14,8 @@ jobs:
     steps:
       - name: Clone the repository
         uses: actions/checkout@v4
+      - name: Install toolchain
+        run: rustup toolchain install 1.69 && rustup +1.69 target add wasm32-unknown-unknown
```

Comment:
Can you bring more context to why is `v1.69`?

Reply:
The nearblocks.io parses the contract by using the [near-contract-parser](https://github.com/NEARFoundation/near-contract-parser), but this tool can't parse contracts that are built with toolchain version 1.7 and over